tools/start_test.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
Okay. I am thinking about passing an argument to run as `./start_tmux.sh test` to indicate that is a test. @saijel